bool req
---
string mode
string mission
